doc/install/index.rst

```diff
@@ -1,72 +1,74 @@
-=======================
- Installation (Manual)
-=======================
+.. _install-overview:
 
+===============
+Installing Ceph
+===============
 
-Get Software
+There are different options to install Ceph. Each solution meet different
+requirements, like ease of maintenance.
+
+
+ceph-ansible
 ============
 
-There are several methods for getting Ceph software. The easiest and most common
-method is to `get packages`_ by adding repositories for use with package
-management tools such as the Advanced Package Tool (APT) or Yellowdog Updater,
-Modified (YUM). You may also retrieve pre-compiled packages from the Ceph
-repository. Finally, you can retrieve tarballs or clone the Ceph source code
-repository and build Ceph yourself.
+Installs Ceph using Ansible.
 
+* `docs.ceph.com/ceph-ansible <https://docs.ceph.com/ceph-ansible/>`_ 
 
-.. toctree::
-   :maxdepth: 1
+ceph-deploy
+===========
 
-	Get Packages <get-packages>
-	Get Tarballs <get-tarballs>
-	Clone Source <clone-source>
-	Build Ceph <build-ceph>
-    	Ceph Mirrors <mirrors>
-	Ceph Containers <containers>
+Install ceph using ceph-deploy
 
+* :ref:`ceph-deploy-index`
```

Comment:
Caveats:
- ceph-deploy is no longer actively maintained.
- ceph-deploy is not tested on versions newer than Nautilus.
- ceph-deploy does not support RHEL/CentOS 8 or newer.